README.md

```diff
@@ -10,10 +10,11 @@ IAM credentials.
 
 ## Using neptune-python-utils
 
-1. Run the docker image passing in the env vars with your "AWS variables" and the Neptune endpoint:
+1. Run the docker image passing in the env vars with your "AWS variables", the Neptune endpoint host and port:
 ```bash
 docker run --rm -ti -p 8888:8888 \
     -e NEPTUNE_ENDPOINT="neptune.endpoint.example.com" \
```

Comment:
Should we rename `NEPTUNE_ENDPOINT` to `NEPTUNE_HOST`?

Reply:
Yes it's a leftover thanks, I'm changing it.